pre_commit/file_classifier/classifier.py

```diff
@@ -0,0 +1,142 @@
+# encoding: utf-8
```

Comment:
herp, can you add the `fix-encoding-pragma` hook from `pre-commit-hooks` and run on all the things?